scripts/build/build.sh

```
@@ -1,35 +1,35 @@
 #!/bin/bash
 set -euo pipefail
 
-# ──────────────────────────────
+# ------------------------------
 # 1) CONFIGURATION
-# ──────────────────────────────
+# ------------------------------
 TARGET_DIRS=("test/driver_asyncpg", "test\driver_aiosqlite", "test\driver_sqlite3")
```

Shellcheck (0.11.0)

[warning] Use spaces, not commas, to separate array elements.

(SC2054)